public/app/plugins/datasource/loki/components/LokiQueryEditor.tsx

```diff
@@ -36,7 +37,7 @@ export const LokiQueryEditor = React.memo<LokiQueryEditorProps>((props) => {
   const [dataIsStale, setDataIsStale] = useState(false);
   const [labelBrowserVisible, setLabelBrowserVisible] = useState(false);
   const [queryStats, setQueryStats] = useState<QueryStats | null>(null);
-  const { flag: explain, setFlag: setExplain } = useFlag(lokiQueryEditorExplainKey);
+  const [explain, setExplain] = useState(window.localStorage.getItem(lokiQueryEditorExplainKey) === 'true');
```

Comment:
I think this is fine for now, I was still wondering if we want to have any `localStorage` abstraction at one point. I think with this, it's just a matter of time until someone uses `window.localStorage.getItem(anyKey) === true` without being noticed. Do you know if there's been any discussions about moving [`Store`](https://github.com/grafana/grafana/blob/main/public/app/core/store.ts) to an accessible place?

Reply:
> I think with this, it's just a matter of time until someone uses window.localStorage.getItem(anyKey) === true without being noticed.

So this would be an issue even if `Store` was used with `store.get`, but it is true that it has `store.getBool`. I don't think there was a discussion, but I will add it to the list of code that could be shared across multiple data sources that we are documenting + figuring out where to store it.